angular/core/app.coffee

```diff
@@ -43,6 +43,9 @@ angular.module('loomioApp', ['ngNewRouter',
 angular.module('loomioApp').controller 'ApplicationController', ($scope, $filter, $rootScope, $router, KeyEventService, ScrollService, CurrentUser, BootService, AppConfig, ModalService, ChoosePlanModal, AbilityService) ->
   $scope.isLoggedIn = AbilityService.isLoggedIn
 
+  if document.location.protocol.match(/https/) && navigator.serviceWorker?
```

Comment:
We're adding a service worker here, not because we need one, but because chrome requires it for your thing to be seen as 'an app'.

One the plus side, now I know a little bit more about workers and how we can leverage them towards nifty things like push notifications in the future.